doc/src/thermo_modify.rst

```diff
@@ -75,6 +76,23 @@ are drifting out of the box through a fixed boundary condition (see
 the :doc:`boundary <boundary>` command).  In this case one atom may be
 deleted before the rest of the molecule is, on a later timestep.
 
+The *warn* keyword determines whether LAMMPS will print warning
+messages and how many of them.  Certain warning messages can be
```

Comment:
I think this should say something like this:
Most warning messages are only printed by processor 0.  These are often important to read.  Some warning messages are printed during a run (or immediately before) by every processor that encouters the issue, e.g. bonds that are stretched too far or dihedrals that in extreme configurations.  These can blow up ...